<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImsStoreMainWindow</class>
 <widget class="QMainWindow" name="ImsStoreMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Receive Reels</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../universal/debtcollection/res.qrc">
    <normaloff>:/images/autoship.ico</normaloff>:/images/autoship.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Incoming Containers:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCommandLinkButton" name="cmdRefreshContainers">
        <property name="text">
         <string>Refresh</string>
        </property>
        <property name="description">
         <string>Refresh the container list:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="CustomQueryView" name="trvContainers"/>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Reels:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tblReels"/>
        </item>
        <item>
         <widget class="QLabel" name="lblSummary">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCommandLinkButton" name="cmdSave">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="description">
             <string>Save the reels' list</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../universal/debtcollection/res.qrc"/>
 </resources>
 <connections/>
</ui>
